Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -50,6 +50,22 @@ class SentryFeedback: NSObject, SentrySerializable {
         return dict
     }
     
+    func dataDictionary() -> [String: Any] {
```

Comment:
`m`: Why is this different from serialize? Can't this be the same?

Reply:
It's a different schema, from the docs. There's extra info in the serialization stuff that isn't even available at this point.

Reply:
Please add the info as a comment.